tests/Rector/ClassMethod/MergeMethodAnnotationToRouteAnnotationRector/Fixture/fixture.php.inc

```diff
@@ -33,6 +41,13 @@ class DefaultController
     public function show($id)
     {
     }
+
+    /**
+     * @Route("/post/{id}", methods={"POST"})
+     */
+    public function post($id)
+    {
+    }
```

Comment:
Could you extract the test fixture to a new file? 
So we have this one separated and easier to track in case of regression in the future :+1: